crates/bevy_gltf/src/loader.rs

```diff
@@ -277,9 +277,11 @@ async fn load_gltf<'a, 'b>(
 
     let mut scenes = vec![];
     let mut named_scenes = HashMap::default();
+    let mut scene_to_nodes = HashMap::default();
     for scene in gltf.scenes() {
         let mut err = None;
         let mut world = World::default();
+        let mut nodes_per_scene = vec![];
```

Comment:
Do we know the number of nodes ahead of time? If so, we could preallocate the size of this.